### ISSUE TITLE
Version command

### ISSUE THREAD

**Issue by ericmarkmartin** [hnarayanan/shpotify#59]
For the sake of debugging and compliance with CLI standards, a `version` command of some sort should be included in the shpotify API.

Possible ideas for the command syntax (multiple different syntaxes can be used):

- `spotify -V`
- `spotify -v`
- `spotify --version`
- `spotify version`

**Comment by tpritc** [comment]
I like `spotify -v` and `spotify --version`.

**Comment by hnarayanan** [comment]
Agree with @tpritc.

**Comment by ericmarkmartin** [comment]
I also agree with @tpritc: they are the most standard and thus the most predictable.

**Comment by ericmarkmartin** [comment]
Essentially I'm debating whether to go with the simple version output (like Python):
```
$ python --version
Python 3.4.3
```

Or something with a license:
```
$ bash --version
GNU bash, version 4.3.46(1)-release (x86_64-apple-darwin14.5.0)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

I'm probably overthinking this, but I would rather just get it right the first time.

**Comment by ericmarkmartin** [comment]
Added PR #64

**Comment by hnarayanan** [comment]
Still reeling from the last flurry of commits and the release. Will get to this in time.
